openhcl/underhill_attestation/src/lib.rs

```diff
@@ -294,9 +294,14 @@ pub async fn initialize_platform_security(
             driver,
         )
         .await
-        .map_err(ErrorInner::RequestVmgsEncryptionKeys)?
+        .map_err(|e| {
+            tracing::error!(
+                attestation_type=?attestation_type,
```

Comment:
Is this needed? `attestation_type` is logged earlier, and the error message of `AttestationErrorInner::RequestVmgsEncryptionKeys` should cover the message part.